lib/harness/pkg.ex

```diff
@@ -24,7 +24,12 @@ defmodule Harness.Pkg do
   """
   @callback links(t()) :: [String.t()]
```

Comment:
```suggestion
  @callback links(t()) :: [Path.t()]
```

and samesies for c:hard_links/1 plox